### PR TITLE
Refactoring README

### PR DESCRIPTION
**Pull request by samhstn**
- Rearranging README to be in chronological order. See #20 
- Removing out of date images and opting for images hosted by github

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,139 +1,120 @@
...
-## Chrome
-Install Chrome and set it as the default browser
-Add the [Vimium Chrome Extension](https://vimium.github.io/)
+This is my OSX setup for Sierra 10.12. I like a simple setup.
 
 ## Applications to download
-**Spectacle** - For quick setting of your application window sizes
 
-**Karabiner** - Faster key repeat and key remapping
+[**Googe Chrome**](https://github.com/shouston3/My-Config/tree/master#googe-chrome) - My browser choice
```
Comment (nelsonic) [inline README.md]: @shouston3 agree that Chrome is go-to browser for several reasons.
but remind me to show you some insanely cool features of https://www.mozilla.org/en-GB/firefox/developer/ 😉

Comment (samhstn) [reply README.md]: Will do, exciting times

File: WHY_VIM.md [hunk 2]
```diff
@@ -1,88 +0,0 @@
-# Why Vim
```
Comment (nelsonic) [inline WHY_VIM.md]: @shouston3 is there a reason for deleting this? 🤔

File: README.md [hunk 3]
```diff
@@ -198,49 +197,25 @@ In a new terminal window run: `redis-cli`
...
-Your terminal should look something like this:
+#### Heroku toolbelt
 
-<img src="imgs/terminal.png" />
 
-[Here](https://github.com/shouston3/My-Config/blob/master/sams-aliases.plugin.zsh) are the functions and aliases I use day to day
+#### Misc
 
-They can be installed with:
+<img src="http://core0.staticworld.net/images/article/2013/10/system-preferences-gallery-100065979-large.png" align="right" height="150px" />
```
Comment (nelsonic) [inline README.md]: @shouston3 tip: if you copy the image an paste it in an issue/PR comment:
image
GitHub will "upload" the image to their CDN (S3) which means there's no risk of it vanishing. :-)

**Review by nelsonic — APPROVED** [review]
@shouston3 simplification/refactor looks great! 🎉

File: WHY_VIM.md (hunk 2 above)
Comment (samhstn) [reply WHY_VIM.md]: Going to include this more concisely in the README, I think I waffle on to much in separate files

Comment (samhstn) [reply WHY_VIM.md]: On second thoughts, I'll keep this in. Thanks for catching this 👍

**Comment by samhstn** [comment]
@RobStallion ready to ship unless you spot any major mistakes

Going to give it another improvement once this is in

**Comment by samhstn** [comment]
@RobStallion Just a reminder to review (and merge) this

**Pull request merged**